### PR TITLE
CIT-27 #time 8h start html and css

### PR DESCRIPTION
**Pull request by codal-kcolaco**

### CODE REVIEW

File: week_1/2_apr/youtube-fe/styles/header.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+.header {
+    height: 55px;
+    display: flex;
```
Comment (abhi-codal) [inline week_1/2_apr/youtube-fe/styles/header.css:3]: @codal-kcolaco Can you please check all the possibles and what come with that?

File: week_1/2_apr/youtube-fe/styles/header.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+.left-section {
+    display: flex;
+    align-items: center;
+}
+
+.hamburger-menu {
+    height: 24px;
+    margin-left: 24px;
```
Comment (abhi-codal) [inline week_1/2_apr/youtube-fe/styles/header.css:15]: @codal-kcolaco also check how we can give margin left right in one line or if any other margin as with padding

File: week_1/2_apr/youtube-fe/styles/header.css (hunk 1 above)
Comment (codal-kcolaco) [reply week_1/2_apr/youtube-fe/styles/header.css:3]: @abhi-codal Yes

File: week_1/2_apr/youtube-fe/styles/header.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    flex: 1;
+    margin-left: 70px;
+    margin-right: 35px;
+    max-width: 400px;
+    display: flex;
+    align-items: center;
+}
+
+.search-bar {
+    flex: 1;
```
Comment (abhi-codal) [inline week_1/2_apr/youtube-fe/styles/header.css:33]: What is use of flex:1? why we use it

File: week_1/2_apr/youtube-fe/styles/header.css (hunk 2 above)
Comment (codal-kcolaco) [reply week_1/2_apr/youtube-fe/styles/header.css:15]: @abhi-codal Yes, will do

File: week_1/2_apr/youtube-fe/styles/header.css [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+.search-button {
+    height: 38px;
+    width: 60px;
+    border-style: solid;
+    border-color: gray;
+    border-width: 1px;
+    margin-right: 20px;
+    border-top-right-radius: 20px;
+    border-bottom-right-radius: 20px;
+    margin-left: -1px;
```
Comment (abhi-codal) [inline week_1/2_apr/youtube-fe/styles/header.css:58]: avoid provided margin in -ve. use only if needed

File: week_1/2_apr/practice/buttons.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```
Comment (abhi-codal) [inline week_1/2_apr/practice/buttons.html:1]: @codal-kcolaco please check all values of input tags and how we can use it with diffrent values to input tag.

File: week_1/2_apr/youtube-fe/styles/header.css (hunk 3 above)
Comment (codal-kcolaco) [reply week_1/2_apr/youtube-fe/styles/header.css:33]: @abhi-codal I did it to make the search bar flexible based on the window size. If we give one item flex: 2, and another item flex: 1, then the first item will take two times the amount of space than another item in the axis.

File: week_1/2_apr/practice/buttons.html (hunk 5 above)
Comment (codal-kcolaco) [reply week_1/2_apr/practice/buttons.html:1]: @abhi-codal  Yes

File: week_1/week_1.txt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+2 APRIL
+    HTML Tags
```
Comment (abhi-codal) [inline week_1/week_1.txt:2]: please provide more info which all tags you have learned

File: week_1/week_1.txt [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+    HTML Structure 
+    Chrome Dev Tools 
+    CSS Basics
```
Comment (abhi-codal) [inline week_1/week_1.txt:5]: Same here and for others please add what all things you have learn in detail like 
- css basic
     - display
     - font-size
     - ...
     - ...
....

File: week_1/week_1.txt (hunk 6 above)
Comment (codal-kcolaco) [reply week_1/week_1.txt:2]: @abhi-codal Yes will update this in next commit.

**Review by abhi-codal — APPROVED** [review]

**Pull request merged**